rpcs3/rpcs3.vcxproj.filters

```diff
@@ -60,6 +60,9 @@
       <Extensions>ui</Extensions>
       <ParseFiles>true</ParseFiles>
     </Filter>
+    <Filter Include="Gui\saves">
```

Comment:
Well, there isn't any "data" there.  It's just a bunch of dialogs.  I would be tempted to go with save_dialogs if you want a more descriptive name.